Source/WebKit/GPUProcess/webrtc/LibWebRTCCodecsProxy.mm

```diff
@@ -88,15 +88,19 @@
 {
     protectedConnection()->removeWorkQueueMessageReceiver(Messages::LibWebRTCCodecsProxy::messageReceiverName());
 
-    protectedWorkQueue()->dispatch([this, protectedThis = WTFMove(refFromConnection)] {
-        assertIsCurrent(workQueue());
-        auto decoders = WTFMove(m_decoders);
+    protectedWorkQueue()->dispatch([weakThis = WeakPtr { this }, refFromConnection = WTFMove(refFromConnection)] {
```

Comment:
@mwyrzykowski Just updated this after you reviewed since it wasn't building before. Want to confirm this looks good too? Thanks!

Reply:
Looks good, thank you!